include/alpaka/queue/cuda_hip/QueueUniformCudaHipRt.hpp

```diff
@@ -238,11 +237,17 @@ namespace alpaka
                 TTask const& task) -> void
             {
                 auto spCallbackSynchronizationData = std::make_shared<CallbackSynchronizationData>();
-                ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(ALPAKA_API_PREFIX(StreamAddCallback)(
+#    if defined(ALPAKA_ACC_GPU_CUDA_ENABLED) && CUDA_VERSION > 9020
+                ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(ALPAKA_API_PREFIX(LaunchHostFunc)(
+                    queue.getNativeHandle(),
+                    uniformCudaHipRtCallback,
+                    pCallbackSynchronizationData.get()));
+#    else ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(ALPAKA_API_PREFIX(StreamAddCallback)(
```

Comment:
```suggestion
#    else
                    ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(ALPAKA_API_PREFIX(StreamAddCallback)(
```
?

Reply:
Yes, deleted the new line by mistake (curiously clang-format didn't fix it).